dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties

```diff
@@ -1801,7 +1802,6 @@ oauth2_clients=OAuth2 clients
 
 #-- Authorities ---------------------------------------------------------------#
 
-F_TRACKED_ENTITY_INSTANCE_SEARCH_IN_ALL_ORGUNITS=Search tracked entity instance across all organisation units
```

Comment:
why we removed this one ?

Reply:
it was twice there, I took the latest one written by Lars...